### DISCUSSION TITLE
Expected memory usage for flux components

### DISCUSSION THREAD

**Discussion by brpaz** [fluxcd/flux2#558] [Q&A] [answered]
Hello.

I had installed flux in a new local Kind cluster, and noticed that source-controller pod is using like 600MB RAM.

It seems a little high. For example, kustomize controller is using around 150MB which seems a lot more reasonable.

Is this the expected behavior?

Thank you.

**Comment by stefanprodan** [comment, marked as answer]
Expected memory usage after bootstrap is 30MB per controller. My guess is that your cluster doesn’t correctly report resource usage.

**Comment by brpaz** [comment]
Well, I was testing using a Kind Cluster on my local machine. It might be related to that.

I have installed flux in a real DigitalOcean Kubernetes Cluster and the memory usage is way lower.  (less than 100 MB).
